### PR TITLE
[integration] add APIs description

### PR DESCRIPTION
**Pull request by TaykYoku**
This PR provide short description of the APIs instance.

related PRs:
#5540

BEGINRELEASENOTES

*docs
NEW: add APIs description

ENDRELEASENOTES

### CODE REVIEW

File: docs/source/AdministratorGuide/Introduction/diraccomponents.rst [hunk 1]
```diff
@@ -4,14 +4,19 @@
...
+The basic DIRAC components are *Services*, *APIs*, *Agents*, and *Executors*.
 
   *Services*
     are passive components listening to incoming client requests and reacting accordingly by
     serving requested information from the *Database* backend or inserting requests on the
     *Database* backend. *Services* themselves can be clients of other *Services* from the same
     DIRAC *System* or from other *Systems*.
 
+  *APIs*
+    `RESTful <http://en.wikipedia.org/wiki/Representational_state_transfer>`_ API endpoints that permits
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Introduction/diraccomponents.rst:16]:
```suggestion
    `RESTful <http://en.wikipedia.org/wiki/Representational_state_transfer>`_ API endpoints that permit
```

File: docs/source/AdministratorGuide/Introduction/diraccomponents.rst [hunk 2]
```diff
@@ -47,5 +52,6 @@ Each *System* and *Setup* instance has a distinct name. The mapping of *Systems*
 *Setups* is described in the Configuration of the DIRAC installation in the "/DIRAC/Setups"
 section.
 
-*ToDo*
-  - image illustrating the structure
+.. image:: ../../_static/setup_structure.png
+   :alt: DIAC setup structure illustration
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Introduction/diraccomponents.rst:56]:
```suggestion
   :alt: DIRAC setup structure illustration
```

File: docs/source/AdministratorGuide/Introduction/diraccomponents.rst [hunk 3]
```diff
@@ -4,14 +4,19 @@
...
-The basic DIRAC components are *Services*, *Agents*, and *Executors*.
+The basic DIRAC components are *Services*, *APIs*, *Agents*, and *Executors*.
 
   *Services*
     are passive components listening to incoming client requests and reacting accordingly by
     serving requested information from the *Database* backend or inserting requests on the
     *Database* backend. *Services* themselves can be clients of other *Services* from the same
     DIRAC *System* or from other *Systems*.
 
+  *APIs*
```
[the remark this answers is not in the document]
Comment (TaykYoku) [reply docs/source/AdministratorGuide/Introduction/diraccomponents.rst:15]: I will try to paraphrase this paragraph.
Today (maximum tomorrow) will do PR with the description of API in the corresponding sections AdministratorGuide and DeveloperGuide.

File: docs/source/AdministratorGuide/Introduction/diraccomponents.rst [hunk 4]
```diff
@@ -4,14 +4,20 @@
...
+    are passive components that listen to incoming client requests and respond accordingly by providing the requested information from the server side.
+    The *APIs* themselves can be *Services* clients. Unlike Service, which uses a fixed API to interact with *Services* clients (POST mydirac.com:8443/<system>/<service>),
+    *APIs* can accept GET, POST, etc. HTTP methods, have an easier query path(e.g.: /auth/jwk) and can take path parameters.
+    This flexibility allows implement the `RESTful <http://en.wikipedia.org/wiki/Representational_state_transfer>`_ API to interact with third-party systems.
+
```
[the remark this answers is not in the document]
Comment (TaykYoku) [reply docs/source/AdministratorGuide/Introduction/diraccomponents.rst:20]: It looks better to me

File: docs/source/AdministratorGuide/Introduction/diraccomponents.rst [hunk 5]
```diff
@@ -47,5 +49,6 @@ Each *System* and *Setup* instance has a distinct name. The mapping of *Systems*
 *Setups* is described in the Configuration of the DIRAC installation in the "/DIRAC/Setups"
 section.
 
-*ToDo*
-  - image illustrating the structure
+.. image:: ../../_static/setup_structure.png
+   :alt: DRIAC setup structure illustration
```
Comment (fstagni) [inline docs/source/AdministratorGuide/Introduction/diraccomponents.rst:53]:
```suggestion
   :alt: DIRAC setup structure illustration
```

**Review by fstagni — DISMISSED** [review]

**Review by fstagni — APPROVED** [review]

**Pull request merged**